### PR TITLE
tensorflow 2 and windowing

### PR DESCRIPTION
**Pull request by cjbattey**
This PR updates Locator to use tensorflow2, adds a windowing function for zarr inputs, and cleans up some legacy elements of the code base. The setup.py script has been updated to reflect the tensorflow switch, and the readme has been expanded with better install instructions and a new section on diagnosing failures.

### CODE REVIEW

**Comment by andrewkern** [comment]
@cjbattey can you rebase this to squash your commits?

File: setup.py [hunk 1]
```diff
@@ -3,23 +3,22 @@
...
-      install_requires=["numpy",
+      install_requires=["tensorflow",
+                        "numpy",
                         "h5py",
                         "scikit-allel",
                         "matplotlib",
-                        "scipy",
-                        "keras",
-                        "tensorflow==1.15.2",
+                        "scipy==1.4.1",
```
Comment (andrewkern) [inline setup.py:19]: do you really want to pin to this version? maybe >=1.4.1?

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ python setup.py install
 
 Users without admin access may be able to install to their user account with:
 ```
-python -m pip install --user git+https://github.com/cjbattey/locator
+python -m pip install --user git+https://github.com/kern-lab/locator
```
Comment (andrewkern) [inline README.md:18]: should be kr-colab

File: setup.py (hunk 1 above)
Comment (cjbattey) [reply setup.py:19]: Huh I was getting an error on my machine asking for this version with tf2 but I don't get it on linux or in a fresh environment, so yes let's just call for the latest scipy

Comment (andrewkern) [reply setup.py:19]: 👍

**Pull request merged**